### PR TITLE
[12.0][MIG] purchase_line_procurement_group

### PR DESCRIPTION
**Pull request by kittiu**

### CODE REVIEW

**Comment by kittiu** [comment]
Done Migrated.

**Comment by kittiu** [comment]
Test green, ready for review.

File: purchase_line_procurement_group/views/purchase.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="purchase_order_form_inherit" model="ir.ui.view">
```
Comment (tbaden) [inline purchase_line_procurement_group/views/purchase.xml]: aesthetic change (optional):
use original id (purchase_order_form)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
If you like an optional change:

File: purchase_line_procurement_group/views/purchase.xml (hunk 1 above)
Comment (kittiu) [reply purchase_line_procurement_group/views/purchase.xml]: Fixed. Thank you.

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**